unicodetools/org/unicode/text/utility/Settings.java

```diff
@@ -25,7 +25,7 @@
 
     public static final String UNICODETOOLS_DIRECTORY =
             CldrUtility.getProperty("UNICODETOOLS_DIR",
-                    SVN_WORKSPACE_DIRECTORY + "unicodetools/unicodetools") + '/';
+        	    CLDRPaths.SVN_DIRECTORY + "unicodetools/unicodetools") + '/';
```

Comment:
Why SVN_DIRECTORY since we are no longer using svn?

Reply:
That's just the name for now; needs to be fixed in CLDR. Filed https://unicode-org.atlassian.net/browse/CLDR-13926